src/server/injected/dragScript.ts

```diff
@@ -0,0 +1,115 @@
+/**
```

Comment:
Let's make this a part of InjectedScript, but install the `dragstart` handler lazily as you do now.